src/catalog/src/system_schema/information_schema/information_memory_table.rs

```diff
@@ -236,6 +236,29 @@ pub(super) fn get_schema_columns(table_name: &str) -> (SchemaRef, Vec<VectorRef>
             ],
             vec![],
         ),
+        STATISTICS => (
+            vec![
+                string_column("TABLE_CATALOG"),
```

Comment:
In fact, we have some info for the `statistics` table, you can check the `key_column_usage` for example:

https://github.com/GreptimeTeam/greptimedb/blob/main/src/catalog/src/system_schema/information_schema/key_column_usage.rs

Reply:
Do you mean I should include all the constants including from the crates lines 41-62 in the Statistics table? thanks

Reply:
The table currently returns empty results. Implementing it just like `key_column_usage` to return actual (and mock) data would be better.

Reply:
I see. I will have to implement the virtual table in a file called statistics.rs and write the corresponding test cases like `key_column_usage`, correct? 

Appreciate the guidance